src/panels/lovelace/cards/hui-glance-card.ts

```diff
@@ -16,6 +16,7 @@ import { fireEvent } from "../../../common/dom/fire_event.js";
 import { HassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
 import { HomeAssistant } from "../../../types.js";
 import { LovelaceCard, LovelaceConfig } from "../types.js";
+import { longPress } from "../common/long-press.ts";
```

Comment:
Can we make a new folder for directives and name it `long-press-directive.ts`

Reply:
Should it be a directive, or should we think bigger?